src/background.js

```diff
@@ -1269,9 +1281,25 @@ function initializeGhosteryModules() {
 			if (globals.JUST_UPGRADED_FROM_7) {
 				conf.enable_ad_block = false;
 				conf.enable_anti_tracking = false;
-				setCliqzModuleEnabled(antitracking, conf.enable_anti_tracking);
-				setCliqzModuleEnabled(adblocker, conf.enable_ad_block);
-				setCliqzModuleEnabled(humanweb, IS_EDGE ? false : conf.enable_human_web);
+				setCliqzModuleEnabled(antitracking, conf.enable_anti_tracking)
+					.catch((err) => {
+						if (err !== true) {
+							log('ANTITRACKING ERROR:', err, conf.enable_anti_tracking);
+						}
+					});
+				setCliqzModuleEnabled(adblocker, conf.enable_ad_block)
+					.catch((err) => {
+						if (err !== true) {
+							log('ADBLOKER ERROR:', err, conf.enable_anti_tracking);
+						}
+					});
+
+				setCliqzModuleEnabled(humanweb, IS_EDGE ? false : conf.enable_human_web)
+					.catch((err) => {
+						if (err !== true) {
+							log('HUMANWEB ERROR:', err, conf.enable_anti_tracking);
+						}
+					});
```

Comment:
Can we make these error checkings a bit more intuitive? Right now it's saying "if error was untrue, then there was an error", which is obviously a contradiction.